tokens/fuelux/icon.json

```diff
@@ -0,0 +1,28 @@
+{
+  "imports": [
+    "./aliases.json"
+  ],
+  "props": {
+    "COLOR_TEXT_ICON_DEFAULT": {
+      "value": "{!BLUE_ROYALE}",
+      "type": "color",
+      "category": "icon-color",
+      "cssProperties": ["fill"],
```

Comment:
Probably a nitpick, but couldn't type, category, and cssProperties all be put into a "global" property like https://github.com/ExactTarget/fuelux-mctheme/blob/81f7d02f2b3471068477f4f2ee0d85f9e03818dc/tokens/fuelux/text.json#L2-L6
instead of repeating it a number of times? But like I said, this is a nitpick and I think is good to go otherwise.

Reply:
Good idea!  I'll make that change.